src/Address/PostalCode.php

```diff
@@ -4,12 +4,31 @@
 
 namespace CultuurNet\UDB3\Address;
 
-use CultuurNet\UDB3\StringLiteral;
+use CultuurNet\UDB3\Model\ValueObject\String\Behaviour\Trims;
```

Comment:
We could define extra traits in the `String\Behaviour` namespace, for example:
- `Equals` which implements `sameValueAs`
- `Converts` which implements `toNative`

Reply:
Also fine indeed!